antenna-csv-license-knowledge-base/src/main/java/org/eclipse/sw360/antenna/knowledgebase/csv/CSVBasedLicenseKnowledgeBase.java

```diff
@@ -48,15 +47,15 @@
     private static final String KEY_OSS = "OpenSource";
     private static final String KEY_DELIVER_SRC = "DeliverSources";
     private static final String KEY_DELIVER_LICENSE = "DeliverLicense";
-    private static final String KEY_CLASSIFICATION = "CoveredByINSTStandardProcess";
+    private static final String KEY_CLASSIFICATION = "Classified";
```

Comment:
What does classified mean here, isn't it something like Classification?

Reply:
Yes is something like Classification. I named it classified because it only knew the values ​​yes, no or empty string from the enum `LicenseClassification`. 
Do you have a better name for the column?